src/components/CarouselSection/CarouselCard/CarouselCard.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react'
+import stl from "./CarouselCardStyle.module.scss"
```

Comment:
Always arrange imports by types (separated by empty lines or not). Improves readability.
I follow this:
Package imports
Component imports
Constant/asset imports
Style imports

Reply:
Noted sir